tools/isobuild/import-scanner.ts

```diff
@@ -789,7 +812,12 @@ export default class ImportScanner {
       // newlyMissing and merge the new identifiers back into
       // this.allMissingModules.
       Object.keys(newlyMissing).forEach(id => {
-        if (has(previousAllMissingModules, id)) {
+        const skipScan = has(previousAllMissingModules, id) &&
+          !isHigherStatus(
+            getParentStatus(newlyMissing[id]),
+            getParentStatus(previousAllMissingModules[id]));
```

Comment:
I see why this is important in large applications, since you have to have a bunch of packages (especially custom local packages) to end up with cross-package imports that mix static and dynamic imports together. Makes sense!